components/post_view/post_info/post_info.jsx

```diff
@@ -9,10 +9,11 @@ import {OverlayTrigger, Tooltip} from 'react-bootstrap';
 import {Posts} from 'mattermost-redux/constants';
 import * as ReduxPostUtils from 'mattermost-redux/utils/post_utils';
 
+import CommentIcon from 'components/common/comment_icon.jsx';
```

Comment:
I don't know why you are moving it here, but is not wrong, but you need to remove the extension, because you are renaming the file to `.tsx`